bin/powershell/check-windows-disk-writeable.ps1

```diff
@@ -0,0 +1,76 @@
+#
+#   check-windows-disk-writeable.ps1
+#
+# DESCRIPTION:
+#   This plugin collects the mounted disks and tests they are writeable.
+#
+# OUTPUT:
+#   plain text
+#
+# PLATFORMS:
+#   Windows
+#
+# DEPENDENCIES:
+#   PowerShell
+#
+# USAGE:
+#   Powershell.exe -NonInteractive -NoProfile -ExecutionPolicy Bypass -NoLogo -File C:\\etc\\sensu\\plugins\\check-windows-disk-writeable.ps1
+#   Powershell.exe -NonInteractive -NoProfile -ExecutionPolicy Bypass -NoLogo -File C:\\etc\\sensu\\plugins\\check-windows-disk-writeable.ps1 -Ignore "A,B" -TestFile '\test.txt'
+#
+# NOTES:
+#
+# LICENSE:
+#   Copyright 2017 sensu-plugins
+#   Released under the same terms as Sensu (the MIT license); see LICENSE for details.
+#
+[CmdletBinding()]
+Param(
+  # Disk letters to ignore, e.g. to skip A: and B:, specify "A,B"
+  [Parameter(Mandatory=$False)]
+  [string]$Ignore,
+
+  # Test file to create on each disk to test it is writeable
+  [Parameter(Mandatory=$False)]
+  [string]$TestFile = "\testfile.txt"
```

Comment:
Agreed little value but thought rather leave it as configurable.  Difference between this as and the `Win32_LogicalDisk` filter is I originally intended only to test local drives but can see a use case for network shares too.